packages/cli/src/cmds/beacon/setThreadPoolSize.ts

```diff
@@ -0,0 +1,31 @@
+import os from "node:os";
+
+/**
+ * Sets the libuv thread pool size for worker threads.  This is a critical
+ * component for effective node operations.  Setting the environment variable
+ * must happen almost at the beginning of startup, BEFORE the worker pool is
+ * created by libuv.
+ *
+ * The trigger for creation of the libuv worker pool is scheduling async work
+ * that will queue for a worker.  An example of things that can trigger that
+ * condition are async reading files from the OS.  Some network operations and
+ * any native modules that utilize async work (like @chainsafe/blst-ts).
+ *
+ * Setting this value higher than the number of cores will not be a benefit
```

Comment:
Love the documentation :heart: 

```suggestion
 * Setting this value higher than the number of logical cores will not be a benefit
```